### PR TITLE
Add support for Vagrant packages

### PR DESCRIPTION
**Pull request by KN4CK3R**
This PR adds support for [HashiCorp Vagrant](https://www.vagrantup.com/) boxes/packages.

grafik

### CODE REVIEW

File: routers/api/packages/api.go [hunk 1]
```diff
@@ -265,6 +266,20 @@ func Routes() *web.Route {
 				r.Delete("/yank", rubygems.DeletePackage)
 			}, reqPackageAccess(perm.AccessModeWrite))
 		})
+		r.Group("/vagrant", func() {
+			r.Group("/authenticate", func() {
+				// r.Post("", vagrant.Authenticate)
```
Comment (6543) [inline routers/api/packages/api.go]: :eyes:

Comment (KN4CK3R) [reply routers/api/packages/api.go]: I kept it as reminder if we want to support the CLI login in future. But for the current (working) implementation it's not needed and can be removed if you'd like to.

Comment (6543) [reply routers/api/packages/api.go]: if you like to let it stay in add more description as you did above ... or rm

Comment (KN4CK3R) [reply routers/api/packages/api.go]:
```suggestion
```

**Review by 6543 — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**